doc/exts/pylint_messages.py

```diff
@@ -105,12 +105,10 @@ def _get_python_code_as_rst(code_path: Path) -> str:
     """
     if not code_path.exists():
         return ""
-    with open(code_path, encoding="utf-8") as f:
-        file_content = f.readlines()
     return f"""\
-.. code-block:: python
-
-{"".join("  " + i for i in file_content)}"""
+.. literalinclude:: /{code_path.relative_to(Path.cwd())}
```

Comment:
`/{code_path.relative_to(Path.cwd())}` is equivalent to `/path/to/pylint/doc/{code_path.relative_to(Path.cwd())}`.

Sphinx doc says:

> The file name is usually relative to the current file’s path. However, if it is absolute (starting with /), **it is relative to the top source directory**.